README.md

```diff
@@ -1,2 +1,194 @@
+  <br/>
+
+  <!-- <h3><b>Microverse README Template</b></h3> -->
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Foga's portfolio website] <a name="about-project"></a>
+
+> This is my second Microverse Project. It is based on how to build my portfolio and make it responsive to various devices
+
+**[Foga's portfolio website]** is a website that is used to test for responsiveness to mobile devices
+
+## 🛠 Built With <a name="HTML and CSS"></a>
+
+### Tech Stack <a name="Front end"></a>
+
+> This is a desk-top browser website.
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
+    <li><a href="#">JS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+### Key Features 
+- **[It has a header for navigation]**
+- **[There is a headline section to know more about me and click on the links to connect with me directly]**
+- **[There is a works section to display my recent works]**
+- **[There is a skills section to showcase the skills I have in terms of programming]**
+- **[There is a form section to write me directly about any questions]**
+- **[The portfolio is mobile and desktop responsive]**
+- **[There is a pop-up window for when you click on the see project button]**
+- **[There is form validation for all input fields]**
+- **[Data are stored in local storage]**
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A browser that supports html5 and css3
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:katfogy/setup-and-mobile-first.git
+```
+-
+
+### Install
+
+Install this project with:
+
+- Clone the code into your machine and run test in an editor of your choice(E.g VScode).
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors">Foga Kater Amos</a>
+
+👤 **Author1**
+
```

Comment:
- [ ] Please since you are the only contributor to this project you can singularize the `Authors` to `Author` and replace `Author1` with your name. :smile: